fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/physical/PhysicalOlapScan.java

```diff
@@ -45,6 +46,7 @@
  */
 public class PhysicalOlapScan extends PhysicalCatalogRelation implements OlapScan {
 
+    public final TableSample tableSample;
```

Comment:
Optional, put it under baseOutputs

Reply:
It's not allowed by our checkstyle rules